src/Components/Checkbox/checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { TouchableOpacity } from 'react-native';
```

Comment:
I'd recommend using the [react native checkbox](https://reactnative.dev/docs/checkbox) component. It inherits props from a View component so it shouldn't be difficult to style. It's also probably more customizable.

Reply:
unfortunately that one got disabled, and I tried to use one from the community library but it was super weird